GeoFire.podspec

```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
```

Comment:
We should probably also upgrade the Firebase dependency to 2.2 while we're at it...